javascript/javascript2/week1/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Javascript2 week1</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h2 id="generator">Spirit animal name generator</h2>
+    <input type="text" id="input" placeholder="Write your name here..."><button id="btn">Create</button>
```

Comment:
Wrapping the input and button inside a `form` element is better. Then, your code listens to the `submit` event of the `form` instead of the click event on the `button`. It also allows the user to get a Spirit Animal by pressing the Enter key. The homework only mentioned a click on the button, so I will not request changes for this. But keep in mind the next time you have to implement interactions with an input (maybe in the JS2 project, who knows 🤷🏼‍♂️)